cpp/cmake/thirdparty/get_raft.cmake

```diff
@@ -61,8 +61,8 @@ endfunction()
 # To use a different RAFT locally, set the CMake variable
 # CPM_raft_SOURCE=/path/to/local/raft
 find_and_configure_raft(VERSION  ${RAFT_VERSION}.00
-        FORK                     ${RAFT_FORK}
-        PINNED_TAG               ${RAFT_PINNED_TAG}
+        FORK                     rhdong
```

Comment:
Question- why is this dependent upon RAFT? what do we still have in raft that is needed for this PR?

Reply:
There is a PR for raft which is required by the feature: https://github.com/rapidsai/raft/pull/2382